Exception groups in [[Python 3.11]] will allow you to group multiple exceptions together. This is useful if you have multiple tasks that may fail independently or you have multiple possible resolutions but need to know why each failed.

**References**
- https://realpython.com/python311-exception-groups/#group-exceptions-with-exceptiongroup